pkgs/by-name/ch/chow-kick/package.nix

```
@@ -95,12 +61,12 @@ stdenv.mkDerivation (finalAttrs: {
     cp ChowKick_artefacts/Release/Standalone/ChowKick  $out/bin
   '';
 
-  meta = with lib; {
+  meta =  {
```

CI caught this small double space!
```suggestion
  meta = {
```